charts/tidb-cluster/values.yaml

```diff
@@ -242,6 +246,10 @@ tikv:
   # Default to false.
   hostNetwork: false
 
+  # Specify the security context of TiKV Pod.
+  # refer to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
```

Comment:
There is not sysctl config example in this page, is it OK to add below link?
https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/

Reply:
This field is not only used to specifying sysctls but the entire pod security context